INSTRUCTION.md

```diff
@@ -0,0 +1,26 @@
+# How to Validate the Application Setup
+
+## Run the bootstrap script:
+
+```
+./bootstrap.sh
+```
+
+## Verify Resources
+Ensure all resources are succesfully created using command:
```

Comment:
There is a typo in the word 'successfully'. It should be corrected to ensure clarity in the instructions.